test/tests/parallel/set1/spam_virus/amavis.bats

```diff
@@ -55,29 +58,25 @@ function teardown_file() {
   assert_success
 }
 
-@test '(Amavis enabled) SA ENV should update Amavis config' {
+@test '(Amavis enabled) ENV should update Amavis config' {
   export CONTAINER_NAME=${CONTAINER1_NAME}
-
   local AMAVIS_DEFAULTS_FILE='/etc/amavis/conf.d/20-debian_defaults'
-  # shellcheck disable=SC2016
-  _run_in_container grep '\$sa_tag_level_deflt' "${AMAVIS_DEFAULTS_FILE}"
+
+  _run_in_container grep 'sa_tag_level_deflt' "${AMAVIS_DEFAULTS_FILE}"
   assert_success
-  assert_output --partial '= 2.0'
+  assert_output --partial 'sa_tag_level_deflt = -5.0;'
 
-  # shellcheck disable=SC2016
-  _run_in_container grep '\$sa_tag2_level_deflt' "${AMAVIS_DEFAULTS_FILE}"
+  _run_in_container grep 'sa_tag2_level_deflt' "${AMAVIS_DEFAULTS_FILE}"
   assert_success
-  assert_output --partial '= 6.31'
+  assert_output --partial '$sa_tag2_level_deflt = 2.0;'
 
-  # shellcheck disable=SC2016
-  _run_in_container grep '\$sa_kill_level_deflt' "${AMAVIS_DEFAULTS_FILE}"
+  _run_in_container grep 'sa_kill_level_deflt' "${AMAVIS_DEFAULTS_FILE}"
   assert_success
-  assert_output --partial '= 10.0'
+  assert_output --partial '$sa_kill_level_deflt = 3.0;'
```

Comment:
I think we've got two different tests here.

One that has been deleted is the default values being checked they were set/applied correctly as per our docs.

While the other ones you've carried over here are for verifying the ENV support to override (_although technically both go through that same flow these days_).

Do you not see value in the two distinct tests anymore? I don't mind either way.

Reply:
I will add a third container that checks the defaults 👍🏼

Reply:
Done in [ffe3370](https://github.com/docker-mailserver/docker-mailserver/pull/3820/commits/ffe33702270358ae12da227b29b722451015f98b)